Application/EdFi.Ods.Common/Configuration/DefaultPageSizeLimitProvider.cs

```diff
@@ -1,23 +1,22 @@
+#if NETSTANDARD
```

Comment:
this is in the wrong spot should be above the namespace.

Reply:
Moved above namespace